objects.go

```diff
@@ -55,6 +57,17 @@ func (table *Table) writeOperation(verb verb, ctx *nftContext, writer io.Writer)
 	fmt.Fprintf(writer, "\n")
 }
 
+func parseTable(words []string) (*Table, error) {
+	tableObj := &Table{}
+	if len(words) > 4 {
+		_, comment := splitComment(strings.Join(words[4:], " "))
+		if comment != nil {
+			tableObj.Comment = comment
+		}
```

Comment:
you should return an error if len(words) > 4 but either (a) there's no comment, or (b) there's anything else besides a comment